<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonGo">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>99</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Поехали!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dX">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dY">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ldX">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Шагов по OX</string>
    </property>
   </widget>
   <widget class="QLabel" name="ldY">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Шагов по OY</string>
    </property>
   </widget>
   <widget class="QLabel" name="lErr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>371</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="maxT">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="minT">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>dY</zorder>
   <zorder>lErr</zorder>
   <zorder>buttonGo</zorder>
   <zorder>dX</zorder>
   <zorder>ldX</zorder>
   <zorder>ldY</zorder>
   <zorder>maxT</zorder>
   <zorder>minT</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
